distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -4,55 +4,44 @@ SETLOCAL
 SET ARGS=%*
 SET DIRNAME=%~dp0%
 
-PUSHD %DIRNAME%\..
+PUSHD %DIRNAME%..
 SET DDF_HOME=%CD%
 POPD
 
-:RESTART
+:restart
 REM Remove the restart file indicator so we can detect later if restart was requested
-IF EXIST "%DIRNAME%\restart.jvm" (
-  DEL "%DIRNAME%\restart.jvm"
-)
+IF EXIST "%DIRNAME%\restart.jvm" DEL "%DIRNAME%\restart.jvm"
 
-REM Get Solr port to run on
-for /f "tokens=2 delims==" %%G in ('findstr /i "^\w*solr.http.port=" %DDF_HOME%\etc\system.properties') do (
-    SET SOLR_PORT=%%G
-)
+REM Get Solr client property
+FOR /f "tokens=2 delims==" %%G in ('findstr /i "^\w*solr.client=" %DDF_HOME%\etc\system.properties') DO SET SOLR_CLIENT=%%G
 
-IF "%SOLR_PORT%" == "" (
-    ECHO Property solr.http.port not found in system.properties. Exiting.
-    EXIT /B
-)
+REM Get Solr start property
+FOR /f "tokens=2 delims==" %%G in ('findstr /i "^\w*start.solr=" %DDF_HOME%\etc\system.properties') DO SET SOLR_START=%%G
```

Comment:
❓ Can you use the get_property.bat script here?

Reply:
yup